src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasement.cs

```diff
@@ -127,7 +132,23 @@ protected void InitializeViewHandler(IElement element, IElementHandler handler,
 #endif
 
 				view.Arrange(new Rect(0, 0, w, h));
-				viewHandler.PlatformArrange(view.Frame);
+
+#if WINDOWS
+				if (viewHandler.PlatformView is SwipeControl swipeControl)
```

Comment:
Can we add a check on here if the `swipeControl` has already been loaded or not? I think we can just add `&& !swipeControl.IsLoaded`